client/components/info-tooltip/style.scss

```diff
@@ -2,3 +2,33 @@
 	cursor: help;
 	color: $gray-dark;
 }
+
+&.tooltip.popover.info-tooltip__container {
+	.popover__inner {
+		background: $gray-dark;
+	}
+
+	.popover__arrow {
+		border-top-color: $gray-dark;
+	}
+
+	.info-tooltip__contents {
```

Comment:
Yes, for specificity. I don't remember exactly why, but if I didn't enclose every rule as a child of `&.tooltip.popover.info-tooltip__container`, some styles weren't applied.